lib/pure/oids.nim

```diff
@@ -44,14 +44,15 @@ proc hexbyte*(hex: char): int =
   else: discard
 
 proc parseOid*(str: cstring): Oid =
-  ## parses an OID.
+  ## Parses an OID.
   var bytes = cast[cstring](addr(result.time))
   var i = 0
   while i < 12:
     bytes[i] = chr((hexbyte(str[2 * i]) shl 4) or hexbyte(str[2 * i + 1]))
     inc(i)
 
 proc oidToString*(oid: Oid, str: cstring) =
+  ## Converts a oid to `str`.
```

Comment:
to `str`, which must have space allocated for 25 elements.

Reply:
Sorry I submit wrong commits